app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -0,0 +1,85 @@
+module UspsInPersonProofing
+  class EnrollmentHelper
+    def save_in_person_enrollment(user, profile, pii)
```

Comment:
Since we don't have any instance properties and we're always just calling methods immediately after instantiating, what would you think about making these static methods?

```suggestion
  class EnrollmentHelper
    def self.save_in_person_enrollment(user, profile, pii)
```

Reply:
Mind if I do this in a separate ticket?

Reply:
not a blocker